samples/firefox-accounts/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,5 @@
     <string name="app_name">FxA Android Demo</string>
     <string name="sign_in">FxA Sign In</string>
     <string name="signed_in">Signed in: %1$s</string>
+    <string name="purge">Purge state</string>
```

Comment:
Should we maybe just call this "sign out" or "logout"?

Reply:
updated to "FxA Log Out"